src/dotnet/commands/dotnet-compile/NativeCompiler.cs

```diff
@@ -22,16 +23,37 @@ public override bool Compile(ProjectContext context, CompilerCommandApp args)
 
             var managedOutput = outputPaths.CompilationFiles.Assembly;
 
+            // Create the library exporter
+            var exporter = context.CreateExporter(args.ConfigValue);
```

Comment:
@davidfowl, yeah the user would put this in the project.json runtimes, if native is needed. See my comment earlier, we don't want to block on this work for now, thats why we have the AppDep package.

But this is a good time to get this conversation started. See #402.